__tests__/components/vocabulary/Vocab.test.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Stanford University see LICENSE for license
```

Comment:
Initially tried to create a feature test but was running into a know [error](https://github.com/jsdom/jsdom/issues/2112) with `jsdom` and navigation so just testing with the component rendered with React Router variables.